manim/config_utils.py

```diff
@@ -333,15 +359,17 @@ def _from_command_line():
         else:
             raise FileNotFoundError(f"Config file {args.config_file} doesn't exist")
     else:
-        script_directory_file_config = os.path.join(os.path.dirname(args.file), 'manim.cfg')
+        script_directory_file_config = os.path.join(
+            os.path.dirname(args.file), "manim.cfg"
+        )
         if os.path.exists(script_directory_file_config):
             config_files.append(script_directory_file_config)
 
 else:
     # In this case, we still need an empty args object.
     args = _parse_cli([], input=False)
     # Need to populate the options left out
-    args.file, args.scene_names, args.output_file = '', '', ''
+    args.file, args.scene_names, args.output_file = "", "", ""
 
 config_parser = configparser.ConfigParser()
 successfully_read_files = config_parser.read(config_files)
```

Comment:
Since this has been converted into a utility, it should live in `utils` and not at the top level.